pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -1196,3 +1197,4 @@ https://github.com/ziglang/zig.vim/,,
 https://github.com/mickael-menu/zk-nvim/,HEAD,
 https://github.com/troydm/zoomwintab.vim/,,
 https://github.com/nanotee/zoxide.vim/,,
+https://github.com/asiryk/auto-hlsearch.nvim/,,
```

Comment:
please keep this list sorted, the update script should do this automatically if you do this
```bash
pkgs/applications/editors/vim/plugins/update.py --add asiryk/auto-hlsearch.nvim --add RRethy/nvim-treesitter-endwise
```
then you can remove the update commits if you want

Reply:
done, thanks!